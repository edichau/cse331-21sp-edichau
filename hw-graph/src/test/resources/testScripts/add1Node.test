# Tests that 1 node can be added

# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1

