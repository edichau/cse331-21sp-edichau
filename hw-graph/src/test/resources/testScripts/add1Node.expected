# Tests that 1 node can be added

# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1

